src/helpers/dom.coffee

```diff
@@ -10,4 +10,12 @@ module.exports = {
     if @matches(el, selector) then el else @closest(el.parentNode, selector)
 
 
+  readBootstrapData: (root = document) ->
+    el = root.querySelector('#tutor-boostrap-data')
```

Comment:
Another slight advantage to the "json in script tag" is that it allows the BE to test that the values they're sending are correct.  Since the data can just be fed back into JSON.parse it's easy to validate.  If it was a javascript var, the BE would need to either boot up a javascript interpreter or fall back on regex to test the data.  [current spec](https://github.com/openstax/tutor-server/blob/cacd3f2f08d2cc22eb5ad59493592c6de2d7b59c/spec/controllers/webview_controller_spec.rb#L38-L48)